client/transaction-builder/src/stdlib.rs

```diff
@@ -106,6 +108,41 @@ pub enum ScriptCall {
+    /// with this `hash` can be successfully sent to the network.
+    ///
+    /// # Parameters
+    /// | Name            | Type         | Description                                                                                     |
+    /// | ------          | ------       | -------------                                                                                   |
+    /// | `dr_account`    | `&signer`    | The signer reference of the sending account of this transaction. Must be the Diem Root signer. |
+    /// | `hash`          | `vector<u8>` | The hash of the script to be added to the script allowlist.                                     |
+    /// | `sliding_nonce` | `u64`        | The `sliding_nonce` (see: `SlidingNonce`) to be used for this transaction.                      |
+    ///
+    /// # Common Abort Conditions
+    /// | Error Category             | Error Reason                                                           | Description                                                                                |
+    /// | ----------------           | --------------                                                         | -------------                                                                              |
+    /// | `Errors::NOT_PUBLISHED`    | `SlidingNonce::ESLIDING_NONCE`                                         | A `SlidingNonce` resource is not published under `dr_account`.                             |
+    /// | `Errors::INVALID_ARGUMENT` | `SlidingNonce::ENONCE_TOO_OLD`                                         | The `sliding_nonce` is too old and it's impossible to determine if it's duplicated or not. |
+    /// | `Errors::INVALID_ARGUMENT` | `SlidingNonce::ENONCE_TOO_NEW`                                         | The `sliding_nonce` is too far in the future.                                              |
+    /// | `Errors::INVALID_ARGUMENT` | `SlidingNonce::ENONCE_ALREADY_RECORDED`                                | The `sliding_nonce` has been previously recorded.                                          |
+    /// | `Errors::REQUIRES_ADDRESS` | `CoreAddresses::EDIEM_ROOT`                                           | The sending account is not the Diem Root account.                                         |
+    /// | `Errors::REQUIRES_ROLE`    | `Roles::EDIEM_ROOT`                                                   | The sending account is not the Diem Root account.                                         |
+    /// | `Errors::INVALID_ARGUMENT` | `DiemTransactionPublishingOption::EINVALID_SCRIPT_HASH`               | The script `hash` is an invalid length.                                                    |
+    /// | `Errors::INVALID_ARGUMENT` | `DiemTransactionPublishingOption::EALLOWLIST_ALREADY_CONTAINS_SCRIPT` | The on-chain allowlist already contains the script `hash`.                                 |
+    AddToScriptAllowList { hash: Bytes, sliding_nonce: u64 },
```

Comment:
I thought @sblackshear removed the ability to modify the allowlist? Also I thought we wouldn't be allowing any additions to this list.

Reply:
Ah, it's still in the legacy ABIs so gets generated here automatically, I'll remove it and regenerate.